Economy.md

```diff
@@ -0,0 +1,93 @@
+currency-symbol-suffix: false
+
+# Set the maximum amount of money a player can have.
+# The amount is always limited to 10 trillion because of the limitations of a java double.
+max-money: 10000000000000
+
+# Set the minimum amount of money a player can have (must be above the negative of max-money).
+# Setting this to 0, will disable overdrafts/loans completely.  Users need 'essentials.eco.loan' perm to go below 0.
+min-money: -10000
+
+# Enable this to log all interactions with trade/buy/sell signs and sell command.
+economy-log-enabled: false
+
+# Enable this to also log all transactions from other plugins through Vault.
+# This can cause the economy log to fill up quickly so should only be enabled for testing purposes!
+economy-log-update-enabled: false
+```
+EssentialsX adds the ability to specify the minimum transaction amount to prevent the dreaded microtransactions:
+```yml
+# Minimum acceptable amount to be used in /pay.
+minimum-pay-amount: 0.001
+```
+EssentialsX adds currency format to improve the localization. You may uncomment the 'currency-symbol-format-locale' to use our preformatted styles or uncomment 'currency-format' and create your own:
+```yml
+# The format of currency, excluding symbols. See currency-symbol-format-locale for symbol configuration.
+#
+# "#,##0.00" is how the majority of countries display currency.
+#currency-format: "#,##0.00"
+
+# Format currency symbols. Some locales use , and . interchangeably.
+# Some formats do not display properly in-game due to faulty Minecraft font rendering.
+#
+# For 1.234,50 use de-DE
+# For 1,234.50 use en-US
+# For 1'234,50 use fr-ch
+#currency-symbol-format-locale: en-US
+```
+#### Permissions
+[Permissions for commands can be found here](https://essinfo.xeya.me/permissions.html) (Community maintained list)
+
+Important permissions include `essentials.eco` and  `essentials.pay`.
+## Signs & Shops
+Signs may be used to create shops where players can buy and sell items/blocks to and from the server in exchange for money. 
+#### Signs
+To create a sign, ensure that you have the `signs.create.trade` permission and that you have uncommented `- buy` and `- sell` under `enabledSigns:` in the config.yml
+Signs follow the format:
+```
+[Buy]/[Sell]
+Quantity
+[Material](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html)
+Cost
+```
+###### Example:
+What is typed into the sign UI | Essentials will turn the `[Buy]`/`[Sell]` blue when done sucessfully
+:-------------------------------:|:-------------------------------:
+| ![Image of Sign Creation UI](/images/EcoBuySignUI.png?v=4&s=360) | ![Image of Completed Buy/Sell Signs](/images/EcoBuySellSign.png?v=4&s=400) |
+
+The sign will allow players with the `signs.use.buy` permission to purchase 1 dirt for $1 when left clicked. If `[Buy]` is changed to `[Sell]` and the player has the respective permission, the player may sell 1 dirt to the server in exchange for $1.
```

Comment:
The signs are already explained on the [old original Essentials wiki](https://wiki.mc-ess.net/wiki/Sign_Tutorial), and it thoroughly explains all the signs already (including `[trade]` which is missing here). I'd prefer it if this section just mentions that `[trade]`, `[buy]` and `[sell]` exist and then linked to that page. We can add our own page for signs later down the line.